lib/daru/dataframe.rb

```diff
@@ -341,11 +345,6 @@ def initialize source, opts={}
       update
     end
 
-    def vector(*)
-      $stderr.puts '#vector has been deprecated in favour of #[]. Please use that.'
-      self[*names]
-    end
-
```

Comment:
ok works. Make a separate commit of this so that I can put it in the History later.